includes/sdk-device-id/change-scenarios.md

```diff
@@ -0,0 +1,8 @@
+#### When does a device ID change
```

Comment:
Do we miss this section for mobile?

Reply:
I feel like only browser SDKs have complicate device id change scenarios. Mobile SDKs only change device id when `setDeviceId()` is called. So didn't include it.

Reply:
Mobile also has the same cases. setDeviceId(), and also unset(), pass device Id thought the config.

Reply:
Mobile SDKs will change the device id if they wipe out the data, setDeviceId, or call reset(). I think the only missing thing compare with the browser is to set the deviceId from the URL.  Mention this just for doc alignment, but I think it's good for me.